src/main/java/com/mail/repository/DB.java

```diff
@@ -0,0 +1,144 @@
+package com.mail.repository;
+
+import com.mail.entities.TextMessage;
+import com.mail.repository.interfac.DBInterface;
+import jakarta.persistence.EntityNotFoundException;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Repository;
+
+import java.sql.*;
```

Comment:
Avoid import all from package.
Because of this when you start typing something you will get enormous amount of false suggestions from IDE.